modules/lazyload.nix

```
@@ -26,5 +27,31 @@ in
           '';
         }
       ];
+    warnings =
+      let
+        ignoredPackages = [
+          # removed
+          "treesitter-playground"
+          # renamed
+          "surround"
+          "null-ls"
+          "wilder-nvim"
+        ];
+
+        pluginsWithLazyLoad = builtins.filter (
+          x:
+          !(lib.elem x ignoredPackages)
+          && lib.hasAttr "lazyLoad" config.plugins.${x}
+          && config.plugins.${x}.lazyLoad.enable
+        ) (builtins.attrNames config.plugins);
+        count = builtins.length pluginsWithLazyLoad;
+      in
+      lib.optionals (count > 0) [
+        ''
+          You have enabled experimental lazy loading support.
```

```suggestion
          [Nixvim] WARNING: You have enabled experimental lazy loading support.
```
Something like this ?

---

I'll add the Nixvim prefix, similar to how we do throughout the codebase. But, I don't know if I want to make it sound scary :P

---

Sure, this is completely fair.